stdlib/public/Concurrency/Executor.swift

```diff
@@ -69,6 +69,18 @@ public struct UnownedSerialExecutor: Sendable {
   }
 }
 
+/// Checks if the current task is running on the expected ``Executor``.
+///
+/// Generally, Swift programs should be constructed such that it is statically
+/// known that a specific executor is used, e.g. by using global actors or
+/// custom executors. however, in some APIs it may be useful to provide an
+/// additional runtime check for this, especially when moving towards Swift
+/// concurrency from other runtimes which frequently use such assertions.
+/// - Parameter executor:
```

Comment:
```suggestion
/// Checks if the current task is running on the expected executor.
///
/// Generally, Swift programs should be constructed such that it is statically
/// known that a specific executor is used, for example by using global actors or
/// custom executors. However, in some APIs it may be useful to provide an
/// additional runtime check for this, especially when moving towards Swift
/// concurrency from other runtimes which frequently use such assertions.
/// - Parameter executor: The expected executor.
```